response_operations_ui/views/messages.py

```diff
@@ -532,6 +517,21 @@ def view_selected_survey(selected_survey):  # noqa: C901
 
         pagination = pagination_processor(tab_counts["current"], limit, page, href)
 
+        if thread_id:
```

Comment:
I would be tempted to one line it (naturally over multiple lines), but being chunky it could be left. I would eyeball how it looked.